### PR TITLE
Release 1.1.0 version and doc updates

### PR DESCRIPTION
**Pull request by adamedx**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -6,74 +6,26 @@ Example Note:
...
 Details about the thing that changed that needs to get included in the Release Notes in markdown.
 -->
-# knife-windows 1.0.0 release notes:
-This release of knife-windows includes new features to improve authentication,
-simplify use of the WinRM SSL transport, install and download Chef
-Client during bootstrap, and addresses compatibility issues with Chef Client 12.0.
+# knife-windows 1.1.0 release notes:
+This release of knife-windows includes an important fix for an
+incompatibility issue with Chef Client 12.5 during bootstrap. If you
+are running knife-windows 1.1.0, please upgrade to this version. See
```
Comment (chefsalim) [inline RELEASE_NOTES.md]: should be "if you are running knife-windows 1.0.0" ?

**Comment by adamedx** [comment]
Thx @chefsalim, I fixed the version error.

**Comment by adamedx** [comment]
@btm, @mwrock, need another look.

**Comment by mwrock** [comment]
:+1:

**Pull request merged**